src/taco-cli/cli/taco.ts

```diff
@@ -107,12 +107,12 @@ class Taco {
                             });
```

Comment:
We'll also have to make sure to return the Taco.runWithParsedArgs promise above now